### PR TITLE
fix bannedFilter

### PR DESCRIPTION
**Pull request by subinps**

### CODE REVIEW

File: userge/plugins/utils/botpm.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 START_MEDIA = os.environ.get("START_MEDIA", None)
 
 botPmFilter = filters.create(lambda _, __, ___: BOT_PM)
-bannedFilter = filters.create(lambda _, __, ___: filters.user(_BANNED_USERS))
+bannedFilter = filters.create(lambda _, __, ___: ___.chat.id in _BANNED_USERS)
```
Comment (Harsh-br0) [inline userge/plugins/utils/botpm.py:50]: Try like this maybe
```suggestion
bannedFilter = filters.chat(_BANNED_USERS)
```

Comment (Harsh-br0) [reply userge/plugins/utils/botpm.py:50]: And yeh , add space around the `=` in [here](https://github.com/UsergeTeam/Userge/blob/663cded63981d832f395489feb426d4243e1592b/userge/plugins/utils/botpm.py#L141)

**Review by Krishna-Singhal — APPROVED** [review]

File: userge/plugins/utils/botpm.py (hunk 1 above)
Comment (subinps) [reply userge/plugins/utils/botpm.py:50]: > Try like this maybe

for some unknown reason it did't worked for me (filters.user also)

Comment (Harsh-br0) [reply userge/plugins/utils/botpm.py:50]: No worries , this should be fine too ig
Thanks for the contribution

**Pull request merged**